lib/App.js

```diff
@@ -1444,13 +1440,114 @@ class App {
 
       await writeFileAsync( appJsonPath, JSON.stringify(appJson, false, 2) );
     }
 
     Log(colors.green(`✓ Driver created in \`${driverPath}\``));
+  }
 
+  async questionDriverWireless({ driverPath, driverJson }) {
+    const { wirelessType } = await inquirer.prompt([
+      {
+        type: 'list',
+        name: 'wirelessType',
+        message: 'What type of device is this?',
+        choices: [
+          { name: "Other", value: "other" },
+          { name: "LAN", value: "lan" },
+          { name: "Infrared", value: "infrared" },
+          { name: "433 Mhz/868 Mhz", value: "radio" },
+          { name: "Zigbee", value: "zigbee" },
+          { name: "Z-Wave", value: "zwave" },
+        ]
+      }
+    ]);
+
+    switch (wirelessType) {
+      case 'infrared': {
+        const { shouldInstallRFDriver } = await inquirer.prompt([
+          {
+            type: 'confirm',
+            name: 'shouldInstallRFDriver',
+            message: 'Do you want to install homey-rfdriver?',
+          }
+        ]);
+    
+        if (shouldInstallRFDriver) {
+          await NpmCommands.install({ save: true, path: this.path }, { id: 'homey-rfdriver' });
+        }
+        return;
+      }
+
+      case 'radio': {
+        const { shouldInstallRFDriver } = await inquirer.prompt([
+          {
+            type: 'confirm',
+            name: 'shouldInstallRFDriver',
+            message: 'Do you want to install homey-rfdriver?',
+          }
+        ]);
+    
+        if (shouldInstallRFDriver) {
+          await NpmCommands.install({ save: true, path: this.path }, { id: 'homey-rfdriver' });
+        }
+        return;
+      }
+
+      case 'zigbee': {
+        const { shouldInstallZigbeeDriver } = await inquirer.prompt([
+          {
+            type: 'confirm',
+            name: 'shouldInstallZigbeeDriver',
+            message: 'Do you want to install homey-zigbeedriver?',
+          }
+        ]);
+    
+        if (shouldInstallZigbeeDriver) {
+          await NpmCommands.install({ save: true, path: this.path }, { id: 'homey-zigbeedriver' });
+        }
+        return;
+      }
+
+      case 'zwave': {
+        const { shouldInstallZwaveDriver } = await inquirer.prompt([
+          {
+            type: 'confirm',
+            name: 'shouldInstallZwaveDriver',
+            message: 'Do you want to install homey-zwavedriver?',
+          }
+        ]);
+    
+        if (shouldInstallZwaveDriver) {
+          await NpmCommands.install({ save: true, path: this.path }, { id: 'homey-zwavedriver' });
+        }
+
+        await ZWave.autocompleteDriver({ driverPath, driverJson });
+        return;
+      }
+
+      case 'lan': {
+        const { createDiscovery } = await inquirer.prompt([
+          {
+            type: 'confirm',
+            name: 'createDiscovery',
+            default: false,
+            message: 'Do you want to create a Discovery strategy to find your device automatically in the IP network?'
```

Comment:
Not sure if "in the IP network" is very nice, maybe "in the LAN"?

Reply:
I agree, when I spotted this I thought about changing it to "local network" but then technically Zigbee and Z-Wave are also local networks. IP network kind of makes sense to differentiate it from Zigbee and Z-wave.

Reply:
https://github.com/athombv/node-homey/pull/173